src/sponge/pad.rs

```diff
@@ -30,73 +31,73 @@ where
 #[cfg(test)]
 mod tests {
     use super::*;
-    use bulletproofs::r1cs::LinearCombination;
+    use dusk_bls12_381::Scalar;
+    use dusk_plonk::constraint_system::StandardComposer;
 
     #[test]
     fn test_scalar_padding_width_3() {
-        let zero = Scalar::zero();
-        let one = Scalar::one();
+        let pad_value = Scalar::zero();
+        let eom_value = Scalar::one();
         let two = Scalar::from(2u64);
         let three = Scalar::from(3u64);
         let four = Scalar::from(4u64);
 
-        assert_eq!(&pad(&[two], 3), &[zero, two, one]);
-        assert_eq!(&pad(&[two, three], 3), &[zero, two, three, zero, one, zero]);
         assert_eq!(
-            &pad(&[two, three, four], 3),
-            &[zero, two, three, zero, four, one]
+            &pad(&[two], 3, pad_value, eom_value),
+            &[pad_value, two, eom_value]
+        );
+        assert_eq!(
+            &pad(&[two, three], 3, pad_value, eom_value),
+            &[pad_value, two, three, pad_value, eom_value, pad_value]
+        );
+        assert_eq!(
+            &pad(&[two, three, four], 3, pad_value, eom_value),
+            &[pad_value, two, three, pad_value, four, eom_value]
         );
     }
 
     #[test]
     fn test_scalar_padding_width_4() {
-        let zero = Scalar::zero();
-        let one = Scalar::one();
+        let pad_value = Scalar::zero();
```

Comment:
Since you used `padder` and `eom` as identifiers later on, I think we should be consistent and use them in the tests as well. Or viceversa. Whatever you like more.